save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/LatestExecutionStatisticDto.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqfn.save.agent
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property testSuiteName
+ * @property countTest
+ * @property countPassedTest
+ */
+@Serializable
+data class LatestExecutionStatisticDto(
+    val testSuiteName: String,
+    val countTest: Int,
+    val countPassedTest: Int,
```

Comment:
You create this object for arbitrary statuses, so please rename this field and also add `status` field here too